### PR TITLE
Features/resize oracle

### PR DESCRIPTION
**Pull request by staccDOTsol**
For much larger games the authority (or permissionless) oracle may want to be resized.

@bhgames at your convenience this would help me with my hackathon submission :)

### CODE REVIEW

**Comment by staccDOTsol** [comment]
in the case of https://hackblock.space:

if the authority can have an orderly object that includes many amounts from many players then the players themselves can take turns at concurrently paying out proceeds from previous rounds. It would also track who has entered when.

Cheers :)

File: js/cli/src/matches.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ import {
   getMatchesProgram,
   Utils,
   State
-} from "@raindrops-protocol/raindrops";
+} from "../../lib/";
```
Comment (long-banana) [inline js/cli/src/matches.ts:15]: Switch this back to using the package

File: rust/matches/src/lib.rs [hunk 2]
```diff
@@ -23,7 +23,7 @@ use metaplex_token_metadata::instruction::{
     mint_new_edition_from_master_edition_via_token, update_metadata_accounts,
 };
 use spl_token::instruction::{initialize_account2, mint_to};
-anchor_lang::declare_id!("mtchsiT6WoLQ62fwCoiHMCfXJzogtfru4ovY8tXKrjJ");
+anchor_lang::declare_id!("9UPxDy3p6LjDck7LJAvCwqCXa8qUCG6oq2feuw7pgvH");
```
Comment (long-banana) [inline rust/matches/src/lib.rs:26]: Switch this back to the original id

File: vanity.MD [hunk 3]
```diff
@@ -0,0 +1 @@
+hm
```
Comment (long-banana) [inline vanity.MD:1]: Remove

File: yarn.lock [hunk 4]
```diff
@@ -2,16 +2,7 @@
 # yarn lockfile v1
```
Comment (long-banana) [inline yarn.lock:2]: This PR shouldn't have any updates to the `yarn.lock` file

File: js/cli/yarn.lock [hunk 5]
```diff
@@ -9,7 +9,7 @@
   dependencies:
```
Comment (long-banana) [inline js/cli/yarn.lock:9]: This PR shouldn't have any updates to the `yarn.lock` file

**Review by long-banana — CHANGES_REQUESTED** [review]
Thanks for submitting this change.

Can you modify your approach to create a separate instruction to handle just resizing an oracle. Since the instruction already handles the ability to either create or update an oracle, adding the ability to also resize it is overloading this instruction with too many responsibilities.

File: rust/matches/src/lib.rs (hunk 2 above)
Comment (staccDOTsol) [reply rust/matches/src/lib.rs:26]: duh lol

**Comment by staccDOTsol** [comment]
what the fck yarn.lock

**Comment by staccDOTsol** [comment]
closing in favor of https://github.com/raindrops-protocol/raindrops/pull/30

**Pull request closed without merging**